src/views/Browse.vue

```diff
@@ -1,10 +1,10 @@
 <template>
-  <section>
+  <section class="browse d-flex flex-column">
     <b-alert v-if="!allowExternalAccess && isExternal">
       <p>Accessing external catalogs is not allowed!</p>
     </b-alert>
     <ErrorAlert v-if="error" :dismissible="false" :url="url" :description="errorDescription" :id="errorId" />
-    <div v-else-if="loading" class="loading text-center">
+    <div v-else-if="loading" class="loading text-center d-flex align-items-center justify-content-center flex-grow-1">
       <b-spinner label="Loading..." />
```

Comment:
Could it make sense to make the spinner a bit larger and maybe gray-600 (as used for h2) instead of black?

Reply:
Actually, in another PR we now added a Loading component so the code got a bit simpler, but doesn't allow for too many custom changes anymore. Let's leave it as it is for now.